proto/quicksilver/interchainstaking/v1/messages.proto

```diff
@@ -109,3 +123,29 @@ message MsgSignalIntent {
 
 // MsgSignalIntentResponse defines the MsgSignalIntent response type.
 message MsgSignalIntentResponse {}
+
+// MsgAddValidatorDenyList represents a message type for adding validator to deny list
+message MsgAddValidatorDenyList {
+  option (gogoproto.equal) = false;
+  option (gogoproto.goproto_getters) = false;
+
+  string chain_id = 1 [(gogoproto.moretags) = "yaml:\"chain_id\""];
```

Comment:
These require a title and description. They actually serve no functional purpose, but all block explorer expect these fields to be present in order to display to the user.

Reply:
Also, from_address should be `authority`

Reply:
Oh I see this exists in proposals. MsgAddValidatorDenyList and MsgRemoveValidatorDenyList shouldn't exist then.